ldes-server-port-admin-rest/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/IsIsomorphic.java

```diff
@@ -19,6 +21,10 @@ public static IsIsomorphic with(Model model) {
 		return new IsIsomorphic(model);
 	}
 
+	public static Model isIsomorphicWith(Model model) {
```

Comment:
Can you use the existing solution for this? This class is already an argumentMatcher which makes it possible to use it as argThat(IsIsomorphic.with(model)). The method you are creating here could be confusing out of context since this also is a Resultmatcher. If you see IsIsomorphic.isIsomorphicWith(model), without reading the actual code, no one would expect that there is a side-effect of 'argThat' being called.

When looking at the interface of this class it is also not clear what the difference is between IsIsomorphic.isIsomorphicWith(model) and IsIsomorphic.with(model). However they are very different.